prql-compiler/src/sql/std.sql.prql

```diff
@@ -38,8 +38,9 @@ let read_parquet = source -> s"read_parquet({source})"
 let read_csv = source -> s"read_csv_auto({source})"
 
 let mul = l r -> null
-let div = l r -> null
-let mod = l r -> null
+let div_i = l r -> s"FLOOR(FLOOR(ABS({l} / {r})) * SIGN({l} / {r}))"
+let div_f = l r -> s"({l} * 1.0 / {r})"
```

Comment:
One downside of s-strings is that if `l` is `5 + 5`, then it doesn't understand precedence, and becomes `(5 + 5 * 1.0 / 2)`, so the `/ 2` only operates on `5` rather than `10`.

I guess we handle this, I can check further to confirm.

Reply:
Actually this comes up in https://github.com/PRQL/prql/pull/2692

```
(started + unfinished * 1.0 / started)
# and
* 1.0 / 1 - (started ...
```

Reply:
We need more parenthesis!

Maybe move the "binding strength" calculation into the prql annotations?.

Reply:
I added one option in https://github.com/PRQL/prql/issues/2694...